io.github.anirbandey1.ChatGPTElectron.yml

```diff
@@ -0,0 +1,63 @@
+app-id: io.github.anirbandey1.ChatGPTElectron
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+rename-desktop-file: chatgpt-electron.desktop
+rename-icon: chatgpt-electron
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # - --filesystem=home
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+arch:
+  - "x86"
```

Comment:
This is not valid in any way possible.

Reply:
Yes, I just checked flatpak-builder threw a warning. So, removed this option

Reply:
fixed it in [commit ] (https://github.com/flathub/flathub/pull/4305/commits/f7cc6c9071a2cb182ea515d727fa9d0760c479f4)